File: JavaScript/exercicioManzandoPg46G.js
function fibonacci(){
    let contador = 1
    let numeroAtual = 1
    let numeroAtecessor = 0
    let soma = 0
    console.log(numeroAtual)
    while(contador < 15){
        soma = numeroAtual + numeroAtecessor
        console.log(soma)
        numeroAtecessor = numeroAtual
        numeroAtual = soma
        contador++
    }
}

File: Java/manzando/exercicioManzandoPg46G.java
package manzando;

public class exercicioManzandoPg46G {
	public static void main(String[] args) {
		int contador = 1, numeroAtual = 1, numeroAtecessor = 0, soma = 0;
			    System.out.println(numeroAtual);
			    while(contador < 15){
			        soma = numeroAtual + numeroAtecessor;
			        System.out.println(soma);
			        numeroAtecessor = numeroAtual;
			        numeroAtual = soma;
			        contador++;
			    }
	}
}
